### ISSUE TITLE
task idea: Remove unused imports from files

### ISSUE THREAD

**Issue by hackerkid** [zulip/zulip-gci#617]
We have a lot of unused import in many python files. Removing them can be a task.

**Comment by rht** [comment]
This had been finished in https://github.com/zulip/zulip/pull/7262 actually. I ran flake8 and grepped for F401, while making sure that the ones needed by type annotation (in Python 2 syntax) are still imported.

**Comment by timabbott** [comment]
@rht did that actually cover the whole codebase?  I would have thought it's only done for the parts of the codebase that we've moved to the Python 3 type syntax...

**Comment by rht** [comment]
Yup, I updated the statement here: https://github.com/zulip/zulip/pull/7262#issue-270583498. This can be quickly re-verified later anyway once the full mypy comments to annotations migration has been completed.
